### ISSUE TITLE
Add Graphical LVM management tool like system-config-lvm

### ISSUE THREAD

**Issue by karthikprabhuvinod** [clearlinux/clr-bundles#187]
Currrently there is no way to perform lvm volume management in Clear linux. Looks like there is a open sourced tool called system-config-lvm which can help with graphical logical volume management

https://github.com/hermlnx/system-config-lvm-1.1.18

**Comment by karthikprabhuvinod** [comment]
@mdhorn: This is probably a good to have. We might have to use it in clr-installer to allow users perform LVM management

**Comment by bryteise** [comment]
@karthikprabhuvinod I don't think that is the right upstream url, looks like a dump from another source.

**Comment by karthikprabhuvinod** [comment]
@bryteise Thanks :). This is the right one : git clone git://git.fedorahosted.org/system-config-lvm.git

**Comment by phmccarty** [comment]
@karthikprabhuvinod I can't seem to clone that repo...

Also, I see from the Fedora package repo that the project is no longer used: https://src.fedoraproject.org/rpms/system-config-lvm/commits/master

**Comment by mdhorn** [comment]
@karthikprabhuvinod I was meaning to look into this one:
https://github.com/storaged-project/blivet-gui
as a potential candidate.

**Comment by karthikprabhuvinod** [comment]
@phmccarty Thanks . I was hoping it was the one as I see everyone seems to have a great familiarity with it

@mdhorn Thanks, a good one. It is feature rich and maybe we can think about replacing gparted with it.
Should I close this open one for blivet?

**Comment by mdhorn** [comment]
> @phmccarty Thanks . I was hoping it was the one as I see everyone seems to have a great familiarity with it
> 
> @mdhorn Thanks, a good one. It is feature rich and maybe we can think about replacing gparted with it.
> Should I close this open one for blivet?

I would just reuse this one and maybe update the title. It's the same goal, just a possibly different tool.

**Comment by karthikprabhuvinod** [comment]
@phmccarty : I have updated the tool itself. Please could you evaluate this?

@mdhorn : Thanks. Looks like the tool is pretty feature rich.
